### DISCUSSION TITLE
Contribution Chinese Translation of PK-Sim Documentation

### DISCUSSION THREAD

**Discussion by Slow-Walker66** [Open-Systems-Pharmacology/Forum#1819] [Feature Requests & Ideas]
Dear all,

To support the growing interest in model-informed drug development and precision dosing, a group of over 20 individuals from universities, pharmaceutical companies, and hospitals across China have collaborated to translate the PK-Sim documentation into Chinese. We are pleased to announce that the translation and proofreading have been completed, and we would like to share this translated version on the forum for free use by anyone interested in learning PK-Sim whose first language is Chinese. In this edition, we have also updated some images to reflect PK-Sim V11.

If you find any translation errors, please do not hesitate to contact me at liuxx132@126.com.
[PK-Sim_Documentation.pdf](https://github.com/user-attachments/files/16620432/PK-Sim_Documentation.pdf)

**Comment by lucasfeng13** [comment]
Thank you for the great work and efforts!

**Comment by SCZ1174** [comment]
A remarkable work!

**Comment by AndreDlm** [comment]
Awesome, thanks a lot!
